apps/api/src/routes/admin.routes.ts

```diff
@@ -3790,5 +3790,178 @@ export function configureAdminRoutes(
    */
   router.post("/rewards/allocate", controller.allocateRewards);
 
+  /**
```

Comment:
Also minor - we could add a response example for 501 status to show what `data` field looks like:

```yaml
example:
  success: false
  error: "Not Implemented"
  message: "This endpoint is stubbed..."
  data:
    requestedCount: 2
    note: "When implemented, this will process..."
```